README.md

```diff
@@ -1,24 +1,26 @@
 # Conduit Connector Google Cloud Storage
 
 ### General
-The Google Cloud Storage connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides a source GCS connectors.
+The Google Cloud Storage connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides source GCS connectors.
 
 ### How to build it
 Run `make`.
 
 ### Testing
 
-Run `make test` to run all the tests. You must set the environment variables (`GCP_ServiceAccount_Key`
+Run `make test` with optional `GOTEST_FLAGS` set to run all the tests. You must set the environment variables (`GCP_ServiceAccount_Key`
 , `GCP_ProjectID`, `GCP_Bucket`)
 before you run all the tests. If not set, the tests that use these variables will be ignored.
 
+Cases/Scenarios which are dependent on GCS Error response and where these can't be reproducible are ignored in the test cases.
+ 
+Integration test cases are written to cover all of the scenarios of snapshot and cdc iterators except when tomb dead as it happens only in the above mentioned case.
```

Comment:
I don't think we should mention tomb in the readme.. a bit too technical

Reply:
ok